org-journal.el

```diff
@@ -403,12 +403,24 @@ This runs once per date, before `org-journal-after-entry-create-hook'.")
 
 (defvar org-journal--search-buffer "*Org-journal search*")
 
-
+(defvar org-journal-dont-mark-entries nil)
```

Comment:
This should probably be a `defcustom`, so we can attach some documentation and a type, right?